testing/tests/DevExpress.ui.widgets.editors/checkbox.tests.js

```diff
@@ -305,7 +305,46 @@ QUnit.test("check state changes on space press", function(assert) {
 
     $element.trigger("focusin");
     keyboard.keyDown("space");
-
     assert.equal(instance.option("value"), true, "value has been change successfully");
+
+});
+
+QUnit.module("events");
```

Comment:
```suggestion
QUnit.module("events", function() {
  // add tests here
});
```
This will make it easier to define module boundaries when using IDE